lhotse/cut.py

```diff
@@ -211,19 +211,33 @@ def trimmed_supervisions(self) -> List[SupervisionSegment]:
         """
         return [s.trim(self.duration) for s in self.supervisions]
 
-    def mix(self, other: 'Cut', offset_other_by: Seconds = 0.0, snr: Optional[Decibels] = None) -> 'MixedCut':
+    def mix(
+        self,
+        other: "Cut",
+        offset_other_by: Seconds = 0.0,
+        snr: Optional[Decibels] = None,
+        preserve_id: Optional[str] = None,
+    ) -> "MixedCut":
         """Refer to :function:`~lhotse.cut.mix` documentation."""
-        return mix(self, other, offset=offset_other_by, snr=snr)
+        return mix(self, other, offset=offset_other_by, snr=snr, preserve_id=preserve_id)
 
-    def append(self, other: 'Cut', snr: Optional[Decibels] = None) -> 'MixedCut':
+    def append(
+        self,
+        other: "Cut",
+        snr: Optional[Decibels] = None,
+        preserve_id: Optional[str] = None,
```

Comment:
Should the type be `bool`, not `Optional[str]`?

Reply:
Yeah for append and mix it allows you to choose which cuts ID to preserve (useful in some applications eg padding from both sides)